cmd/nodecmd/wiz.go

```diff
@@ -247,9 +249,32 @@ func wiz(cmd *cobra.Command, args []string) error {
 	} else {
 		ux.Logger.PrintToUser(logging.Green.Wrap("Devnet %s is successfully created and is now validating subnet %s!"), clusterName, subnetName)
 	}
+
+	if err = deployClusterYAMLFile(clusterName, subnetName); err != nil {
+		return err
+	}
+	ux.Logger.PrintToUser("Successfully deployed cluster information YAML file to external host!")
+	ux.Logger.PrintToUser("Cluster information YAML file can be found at /home/ubuntu/clusterInfo.yaml at external host")
 	return nil
 }
 
+func deployClusterYAMLFile(clusterName, subnetName string) error {
+	separateHostInventoryPath := filepath.Join(app.GetAnsibleInventoryDirPath(clusterName), constants.MonitoringDir)
+	separateHosts, err := ansible.GetInventoryFromAnsibleInventoryFile(separateHostInventoryPath)
+	if err != nil {
+		return err
+	}
+	subnetID, chainID, err := getDeployedSubnetInfo(subnetName)
+	if err != nil {
+		return err
+	}
+	if err := createClusterYAMLFile(clusterName, subnetID, chainID, separateHosts[0]); err != nil {
+		return err
+	}
+	ux.Logger.PrintToUser("Deploying cluster information YAML file to external host %s", separateHosts[0].GetCloudID())
```

Comment:
nit: no need to show this as this will be superfast and we will just report result of it